cmd/zoekt-sourcegraph-indexserver/debug.go

```diff
@@ -147,6 +151,55 @@ func debugListIndexed() *ffcli.Command {
 	}
 }
 
+func debugQueue() *ffcli.Command {
+	fs := flag.NewFlagSet("debug queue", flag.ExitOnError)
+
+	printHeader := fs.Bool("header", true, "print column headers")
+
+	connectionTimeout := fs.Duration("timeout", 10*time.Second, "max timeout for establishing a connection to the zoekt-sourcegraph-indexserver instance")
```

Comment:
non-blocking design: do you imagine people actually using these flags in practice? eg to remove a headers from cli output you just do `| tail -n +2`. For connection timeout you just hit ctrl-c or just use the timeout command.